### PR TITLE
TrustSwap decentralized exchange / TrustEVM ✨

### PR DESCRIPTION
**Pull request by stebansaa**
Go EOS!

### CODE REVIEW

**Review by DenisCarriere — APPROVED** [review]
### Product

- [x] Functional product https://trustswap-testnet.web.app
  - Swap Token (EVM<>USDT)
  - Add/Remove liquidity
  - Claim Farm yield
- [x] open source code (most of the code are existing EVM open source forks)
- [x] Esteban (& team) are technically capable to deliver on proposed milestones.

### Token

- [x] No token sale, no airdrop

### Comments

- Great work on current TrustSwap prototype! Glad to see the rapid & active development.

**Review by larryk85 — APPROVED** [review]
It appears that a lot of work has already been done, which is great.

I see no reason to hold up this particular grant.

**Comment by tedcahalleos** [comment]
Congrats! Your Application has been approved by the Grant Evaluators.  You may now proceed onto coding and submit your milestones in the [grant-milestones](https://github.com/eosnetworkfoundation/grant-milestones) repository.

**Comment by stebansaa** [comment]
> Congrats! Your Application has been approved by the Grant Evaluators. You may now proceed onto coding and submit your milestones in the [grant-milestones](https://github.com/eosnetworkfoundation/grant-milestones) repository.

A big thank you to the Eos Network Foundation for the support.

File: applications/trustswap.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+- While we are very early on our development process we have deployed several smart contracts and their respective interfaces. 
+- https://trustswap-testnet.web.app/ is the frontend to our decentralized exchange. It curently makes use of the TrustEVM testnet.
+- https://trustswap-farm.web.app/ is our farm contract, it accepts LP deposits from our decentralized exchange and pays out testnet token. 
+- https://docs.trustswap-farm.web.app/ is our documentation site, it includes general information of our platform. We are using Gitbook for it, will be connected once our domain name is decided.
```
[the remark this answers is not in the document]
Comment (stebansaa) [reply applications/trustswap.md:35]: The certificate issue for the docs subdomain should be fixed once we complete our domain registration. All other staging subdomains appear to be working correctly.

**Pull request merged**